src/components/admin/Login.jsx

```diff
@@ -54,6 +51,12 @@ export default class Login extends BaseComponent {
     }, 100);
   }
 
+  onSignInFailure = async response => {
```

Comment:
Was this outside before just due to programmer error?

Reply:
I guess it was because it didn't use `this`. Actually AirBnB's eslint config enforced this when I upgraded, I disabled it though as I think it's nice having the relevant methods inside

Reply:
Now it uses `this` so then it definitely has to be inside though